mimic/rest/maas_api.py

```diff
@@ -354,6 +359,31 @@ def parse_and_flatten_qs(url):
     return flat_qs
 
 
+def _random_hipsum():
```

Comment:
This should probably be in `mimic/util/helper.py` since text-generation isn't really specific to MaaS.